src/components/Nav/Nav.js

```diff
@@ -1,7 +1,83 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { IoIosArrowDown } from 'react-icons/io';
+
+// COMPONENTS
+import Modal from './Modal/Modal';
+import UserMy from './UserMy/UserMy';
+
+// DATA
+import { LOGIN_INFO, SIGNUP_INFO } from './NavData';
+
+// STYLES
+import * as S from './NavEle';
 
 function Nav() {
-  return <div></div>;
+  const [isModalOpen, SetIsModalOpen] = useState(false);
```

Comment:
```suggestion
  const [isModalOpen, setIsModalOpen] = useState(false);
```
대문자로 시작하는 변수는 컴포넌트를 가리킵니다